web/components/sections/GatedDocumentForm.js

```diff
@@ -0,0 +1,273 @@
+  fieldLabel: classNames("block font-medium"),
+  textField: classNames(
+    "mt-1 block w-full text-base",
+    "border border-gray-300",
+    "focus:ring-indigo-500 focus:border-indigo-500"
+  ),
+};
+
+const serializeForm = function (form) {
+  const returnObject = {};
+  const formData = new FormData(form);
+  for (let key of formData.keys()) {
+    returnObject[key] = formData.get(key);
+  }
+  return returnObject;
+};
+
+function GatedDocumentForm({
+  description,
+  emailTo,
+  image,
+  returnHomeButtonText,
+  submitButtonText,
+  title,
+}) {
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  const formContainerRef = useRef(null);
+  const formRef = useRef(null);
+
+  return (
+    <>
+      <div
+        className={classNames(
+          "max-w-7xl mx-auto px-4 py-24 text-dark-gray",
+          "lg:py-36"
+        )}
+      >
+        <h1
+          className={classNames(
+            "text-5xl text-center leading-tight font-semibold font-sans text-gold pb-24",
+            "3xl:text-7xl",
+            "4xl:text-8xl"
+          )}
+        >
+          {title}
+        </h1>
+        <div
+          className={classNames(
+            "grid grid-cols-1 max-w-7xl mx-auto px-4 text-dark-gray",
+            "md:grid-cols-2"
+          )}
+        >
+          <div className={classNames("max-w-lg text-lg", "lg:text-xl")}>
+            <div className="pb-12">
+              <SanityImage image={image} />
+            </div>
+            <SimpleBlockContent blocks={description} />
+          </div>
+          <div ref={formContainerRef}>
+            {isSubmitted && (
+              <div
+                className="my-12 grid items-stretch border-b border-gold"
+                style={{ height: `${formRef?.current?.offsetHeight}px` }}
+              >
+                <div>
+                  <h2 className="text-3xl font-bold text-gold">Thank you!</h2>
+                  <p className="my-8 text-lg">
+                    We have received your message and will be in touch shortly.
+                  </p>
+                  <Link href="/">
+                    <a className="standardButton">{returnHomeButtonText}</a>
+                  </Link>
+                </div>
+                <div className="min-h-[200px] flex items-center grow justify-end">
+                  {/* eslint-disable-next-line @next/next/no-img-element */}
+                  <img
+                    alt="Brand bars"
+                    height="30px"
+                    src="/images/brand-bars.svg"
+                    width="180px"
+                  />
+                </div>
+              </div>
+            )}
+            {!isSubmitted && (
+              <form
+                className={classNames(isSubmitting && "opacity-50")}
+                ref={formRef}
+                onSubmit={async (event) => {
+                  event.preventDefault();
+
+                  // setIsSubmitting(true);
```

Comment:
Is there a reason this wouldn't be set to true here?

Reply:
None, me being sloppy. Changing now.